<launch>

    <node pkg="aa241x_student" type="vision_node" name="vision_node" output="screen">
        <param name="frame_width" value="640" />
        <param name="frame_height" value="480" />
        <param name="display_video" value="true" />
    </node>

    <node pkg="aa241x_student" type="display_node" name="display_node" output="screen">
        <param name="frame_width" value="640" />
        <param name="frame_height" value="480" />
    </node>



</launch>
